web/app.js

```diff
@@ -224,6 +224,22 @@ const PDFViewerApplication = {
     } catch (ex) {
       console.error(`initialize: "${ex.message}".`);
     }
+
+    if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) {
+      let mode;
+      switch (AppOptions.get("viewerCssTheme")) {
+        case 1:
+          mode = "is-light";
+          break;
+        case 2:
+          mode = "is-dark";
+          break;
+      }
+      if (mode) {
+        document.documentElement.classList.add(mode);
+      }
+    }
+
     if (AppOptions.get("pdfBugEnabled")) {
       await this._parseHashParams();
     }
```

Comment:
To avoid a future issue, let's ensure that *all* options have actually initialized as expected before running the theme-switching code; i.e. please do:
```suggestion
    if (AppOptions.get("pdfBugEnabled")) {
      await this._parseHashParams();
    }

    if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) {
      let mode;
      switch (AppOptions.get("viewerCssTheme")) {
        case 1:
          mode = "is-light";
          break;
        case 2:
          mode = "is-dark";
          break;
      }
      if (mode) {
        document.documentElement.classList.add(mode);
      }
    }
```